### PR TITLE
#166265826 Set up and integrate Travis with README badge

### PR DESCRIPTION
**Pull request by encodedBicoding**
### What does this PR do?
Configures `.travis.yml` file to include scripts needed for travis to run. 
#### Description of Task to be completed?
1. Add git repo to travis using the GUI.
2. Create a `.travis.yml` file with configuration scripts in it.
3. Add travis badge to README.
#### How should this be manually tested?
N/A
#### Any background context you want to provide?
This may not work properly now, as coverage script isn't available yet. 
Coverage script would be available
Screenshot (52)
 when it has been set up by the assigned developer and merged.
#### What are the relevant pivotal tracker stories?
[#166265826](https://www.pivotaltracker.com/story/show/166265826)
#### Questions:
N/A

### CODE REVIEW

File: .travis.yml
Comment (chuxmykel) [inline .travis.yml]: Is this line really necessary?

**Review by chuxmykel — DISMISSED** [review]
I think this PR should wait for @cvjude's PR to be merged first so the tests can reflect

File: .travis.yml
Comment (encodedBicoding) [reply .travis.yml]: Which exactly?

Comment (encodedBicoding) [reply .travis.yml]: i thought i should add it there since we are going to be using the ORM. Correct me if i am wrong

Comment (chuxmykel) [reply .travis.yml]: [Check this out](https://docs.travis-ci.com/user/database-setup/#postgresql)

**Review by encodedBicoding — COMMENTED** [review]
Done. Removed services key

**Comment by emp-daisy** [comment]
Rebase and fix conflicts

File: package.json
Comment (cvjude) [inline package.json:9]: I think we have to change this script before we merge
It's running the tests twice
And its complaining that we don't have coveralls installed.

Comment (encodedBicoding) [reply package.json:9]: I'm on it

Comment (encodedBicoding) [reply package.json:9]: is that supposed to be my job? 
I didn't work on tests

**Review by OvieMudi — APPROVED** [review]

**Review by Hadeneekeh — APPROVED** [review]

**Review by PelumiAlesh — APPROVED** [review]

**Pull request merged**
